error-handling/versions.lock

```diff
@@ -1,13 +1,14 @@
 {
     "compileClasspath": {
         "com.fasterxml.jackson.core:jackson-annotations": {
-            "locked": "2.6.7",
+            "locked": "2.9.5",
             "transitive": [
-                "com.fasterxml.jackson.core:jackson-databind"
+                "com.fasterxml.jackson.core:jackson-databind",
+                "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
```

Comment:
in 2.8 or 2.9 a lot of this stuff got renamed, no? specifically, why don't we have to update `extras/jackson-support/build.gradle` to reflect the new maven coordinates of the Guava and Java8 data formats?